src/profile-logic/gecko-profile-versioning.js

```diff
@@ -1026,5 +1026,69 @@ const _upgraders = {
     }
     convertToVersion20Recursive(profile);
   },
+  [21]: profile => {
+    // Migrate DOMEvent markers to Markers 2.0
+
+    // This is a fairly permissive type, but helps ensure the logic below is type checked.
+    type DOMEventPayload20_to_21 = {
+      // Tracing -> DOMEvent
+      type: 'tracing' | 'DOMEvent',
+      category: 'DOMEvent',
+      // These are removed:
+      eventType: string,
```

Comment:
I think we are keeping the `eventType` still but removing the `phase` and `interval` instead (from looking at the type diff on the previous commit). Is this correct?
We are not removing it on the code actually, so it's safe but we can correct the type here too.